dotnet-desktop-guide/framework/wpf/controls/how-to-create-a-control-that-has-an-access-key-and-text-wrapping.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Create a Control That Has an Access Key and Text Wrapping"
+description: Learn how to create a control that has an access key and text wrapping by means of the included code example in XAML.
```

Comment:
I think you should put a comma after "wrapping" just so it has the same punctuation/construction as your other descriptions.

Reply:
Addressed.